migrations/0001.sqlcode.sql

```diff
@@ -132,7 +132,4 @@ grant execute on sqlcode.CreateCodeSchema to [sqlcode-deploy-role];
 grant execute on sqlcode.DropCodeSchema to [sqlcode-deploy-role];
 grant create procedure to [sqlcode-deploy-role];
 grant create function to [sqlcode-deploy-role];
-grant create type to [sqlcode-deploy-role];
-
-
-alter user
```

Comment:
The `alter user` by itself is not valid SQL.  I'm not sure why it was here in the first place.  My feeling is that this file was never properly tested/migrated before.